ansible/environments/distributed/group_vars/all

```diff
@@ -70,3 +70,4 @@ instances:
   - name: db
     num_instances: 1
     flavor:
+    volume:
```

Comment:
@kkbankol-ibm agreed this is a tricky part. off the top of my head I see two options to solve this:
1. we could work with a template of the hosts file that only describes the groups and how many servers there are in each group. the playbook would read the template, provision the VMs and then generate the actual hosts file from the template and fill in the IPs. I mean the playbook at the moment pretty much does exactly that: it fills the header, then adds the hosts, then puts the registry entry at the bottom - it writes a template.
For reference you could check out the hosts template we use for the mac env. It finds out the IP of the docker-machine VM and fills it in.

2. A separate approach could be to not use a statically generated template at all. I haven't tried it yet but I know if you put in a shell or python script in the environment dir, it will be called to produce a JSON array of hosts. there are plenty of pre-canned inventory scripts already there - OpenStack included.
Check this out http://docs.ansible.com/ansible/intro_dynamic_inventory.html#example-openstack-external-inventory-script

Other than that, I did approve so you are good to merge. Just open a secondary issue to tackle the design changes and we're good.